node/src/components/deploy_acceptor/tests.rs

```diff
@@ -1279,6 +1291,20 @@ async fn should_reject_deploy_with_mangled_transfer_amount() {
     ))
 }
 
+#[tokio::test]
+async fn should_reject_expired_deploy_from_client() {
+    let test_scenario = TestScenario::ShouldNotAcceptExpiredDeploySentByClient;
```

Comment:
I'm too late to the party with my nit, but... :)
I'd prefer consistent naming, either:

```
should_reject_expired_deploy_from_client
ShouldRejectExpiredDeploySentByClient
```

or

```
should_not_accept_expired_deploy_from_client
ShouldNotAcceptExpiredDeploySentByClient
```